src/evm-config.js

```diff
@@ -189,33 +170,23 @@ function sanitizeConfig(name, overwrite = false) {
     config.gen.args.find(arg => arg.includes('cc_wrapper'))
   ) {
     config.gen.args = config.gen.args.filter(arg => !arg.includes('cc_wrapper'));
-
-    if (!overwrite) {
-      console.warn(
-        `${
-          color.warn
-        } Your evm config ${configName} has goma enabled but also defines a ${color.config(
-          'cc_wrapper',
-        )} argument - we've temporarily removed it for you`,
-      );
-    }
+    changes.push(`removed a ${color.config('cc_wrapper')} definition because goma is enabled`);
   }
 
   if (!config.env || !config.env.CHROMIUM_BUILDTOOLS_PATH) {
     const toolsPath = path.resolve(config.root, 'src', 'buildtools');
     config.env.CHROMIUM_BUILDTOOLS_PATH = toolsPath;
-
-    if (!overwrite) {
-      console.warn(
-        `${color.warn} Your evm config ${configName} has not defined ${color.config(
-          'CHROMIUM_BUILDTOOLS_PATH',
-        )} - we've temporarily added it for you`,
-      );
-    }
+    changes.push(`added ${color.config('CHROMIUM_BUILDTOOLS_PATH')} definition`);
   }
 
-  if (overwrite) {
-    save(name, config);
+  if (changes.length !== 0) {
```

Comment:
(nit) for clarity's sake this should probably be `changes.length > 0`

Reply:
changed d70c657